.storybook/docs-root.css

```diff
@@ -1,24 +1,26 @@
-#docs-root .sbdocs-content {
+#storybook-docs .sbdocs-content {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32018&commitId=0f53f6ec76b9c0eb78327f93a883261166a66d4d&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.Accordion With Custom Icons Dark Mode.chromium.png|0| Added|
|Accordion.Accordion With Custom Icons RTL.chromium.png|0| Added|
|Accordion.Accordion With Custom Icons.chromium.png|0| Added|
|Accordion.Disabled Dark Mode.chromium.png|0| Added|
|Accordion.Disabled RTL.chromium.png|0| Added|
|Accordion.Disabled.chromium.png|0| Added|
|Accordion.Marker Position End Dark Mode.chromium.png|0| Added|
|Accordion.Marker Position End RTL.chromium.png|0| Added|
|Accordion.Marker Position End.chromium.png|0| Added|
|Accordion.Size Dark Mode.chromium.png|0| Added|
|Accordion.Size RTL.chromium.png|0| Added|
|Accordion.Size.chromium.png|0| Added|
|Accordion.Accordion With Custom Icons - Dark Mode.normal.chromium.png|0| Removed|
|Accordion.Accordion With Custom Icons - RTL.normal.chromium.png|0| Removed|
|Accordion.Accordion With Custom Icons.normal.chromium.png|0| Removed|
|Accordion.Disabled - Dark Mode.normal.chromium.png|0| Removed|
|Accordion.Disabled - RTL.normal.chromium.png|0| Removed|
|Accordion.Disabled.normal.chromium.png|0| Removed|
|Accordion.Marker Position End - Dark Mode.normal.chromium.png|0| Removed|
|Accordion.Marker Position End - RTL.normal.chromium.png|0| Removed|
|Accordion.Marker Position End.normal.chromium.png|0| Removed|
|Accordion.Size - Dark Mode.normal.chromium.png|0| Removed|
|Accordion.Size - RTL.normal.chromium.png|0| Removed|
|Accordion.Size.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Avatar</b> 38 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Active Appearance Dark Mode.chromium.png|0| Added|
|Avatar.Active Appearance.chromium.png|0| Added|
|Avatar.Active Dark Mode.chromium.png|0| Added|
|Avatar.Active.chromium.png|0| Added|
|Avatar.Color Dark Mode.chromium.png|0| Added|
|Avatar.Color RTL.chromium.png|0| Added|
|Avatar.Color.chromium.png|0| Added|
|Avatar.Colorful Dark Mode.chromium.png|0| Added|
|Avatar.Colorful RTL.chromium.png|0| Added|
|Avatar.Colorful.chromium.png|0| Added|
|Avatar.Custom Initials RTL.chromium.png|0| Added|
|Avatar.Custom Initials.chromium.png|0| Added|
|Avatar.Default RTL.chromium.png|0| Added|
|Avatar.Default.chromium.png|0| Added|
|Avatar.Icon Dark Mode.chromium.png|0| Added|
|Avatar.Icon.chromium.png|0| Added|
|Avatar.Image.chromium.png|0| Added|
|Avatar.Shape.chromium.png|0| Added|
|Avatar.Size.chromium.png|0| Added|
|Avatar.Active - Dark Mode.normal.chromium.png|0| Removed|
|Avatar.Active Appearance - Dark Mode.normal.chromium.png|0| Removed|
|Avatar.Active Appearance.normal.chromium.png|0| Removed|
|Avatar.Active.normal.chromium.png|0| Removed|
|Avatar.Color - Dark Mode.normal.chromium.png|0| Removed|
|Avatar.Color - RTL.normal.chromium.png|0| Removed|
|Avatar.Color.normal.chromium.png|0| Removed|
|Avatar.Colorful - Dark Mode.normal.chromium.png|0| Removed|
|Avatar.Colorful - RTL.normal.chromium.png|0| Removed|
|Avatar.Colorful.normal.chromium.png|0| Removed|
|Avatar.Custom Initials - RTL.normal.chromium.png|0| Removed|
|Avatar.Custom Initials.normal.chromium.png|0| Removed|
|Avatar.Default - RTL.normal.chromium.png|0| Removed|
|Avatar.Default.normal.chromium.png|0| Removed|
|Avatar.Icon - Dark Mode.normal.chromium.png|0| Removed|
|Avatar.Icon.normal.chromium.png|0| Removed|
|Avatar.Image.normal.chromium.png|0| Removed|
|Avatar.Shape.normal.chromium.png|0| Removed|
|Avatar.Size.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Badge</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge.Appearance Dark Mode.chromium.png|0| Added|
|Badge.Appearance.chromium.png|0| Added|
|Badge.Color Dark Mode.chromium.png|0| Added|
|Badge.Color.chromium.png|0| Added|
|Badge.Default Dark Mode.chromium.png|0| Added|
|Badge.Default.chromium.png|0| Added|
|Badge.Shape Dark Mode.chromium.png|0| Added|
|Badge.Shape.chromium.png|0| Added|
|Badge.Size Dark Mode.chromium.png|0| Added|
|Badge.Size.chromium.png|0| Added|
|Badge.Appearance - Dark Mode.normal.chromium.png|0| Removed|
|Badge.Appearance.normal.chromium.png|0| Removed|
|Badge.Color - Dark Mode.normal.chromium.png|0| Removed|
|Badge.Color.normal.chromium.png|0| Removed|
|Badge.Default - Dark Mode.normal.chromium.png|0| Removed|
|Badge.Default.normal.chromium.png|0| Removed|
|Badge.Shape - Dark Mode.normal.chromium.png|0| Removed|
|Badge.Shape.normal.chromium.png|0| Removed|
|Badge.Size - Dark Mode.normal.chromium.png|0| Removed|
|Badge.Size.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Button</b> 192 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button.Circular.chromium.png|0| Added|
|Button.Default.chromium.png|0| Added|
|Button.Large Icon Only Dark Mode.chromium.png|0| Added|
|Button.Large Icon Only.chromium.png|0| Added|
|Button.Large With Icon Dark Mode.chromium.png|0| Added|
|Button.Large With Icon RTL.chromium.png|0| Added|
|Button.Large With Icon.chromium.png|0| Added|
|Button.Large.chromium.png|0| Added|
|Button.Medium Icon Only Dark Mode.chromium.png|0| Added|
|Button.Medium Icon Only.chromium.png|0| Added|
|Button.Medium With Icon Dark Mode.chromium.png|0| Added|
|Button.Medium With Icon RTL.chromium.png|0| Added|
|Button.Medium With Icon.chromium.png|0| Added|
|Button.Medium.chromium.png|0| Added|
|Button.Outline Dark Mode.chromium.png|0| Added|
|Button.Outline Disabled Dark Mode.chromium.png|0| Added|
|Button.Outline Disabled Focusable Dark Mode.chromium.png|0| Added|
|Button.Outline Disabled Focusable.chromium.png|0| Added|
|Button.Outline Disabled.chromium.png|0| Added|
|Button.Outline.chromium.png|0| Added|
|Button.Primary Dark Mode.chromium.png|0| Added|
|Button.Primary Disabled Dark Mode.chromium.png|0| Added|
|Button.Primary Disabled Focusable Dark Mode.chromium.png|0| Added|
|Button.Primary Disabled Focusable.chromium.png|0| Added|
|Button.Primary Disabled.chromium.png|0| Added|
|Button.Primary.chromium.png|0| Added|
|Button.Rounded.chromium.png|0| Added|
|Button.Small Icon Only Dark Mode.chromium.png|0| Added|
|Button.Small Icon Only.chromium.png|0| Added|
|Button.Small With Icon Dark Mode.chromium.png|0| Added|
|Button.Small With Icon RTL.chromium.png|0| Added|
|Button.Small With Icon.chromium.png|0| Added|
|Button.Small.chromium.png|0| Added|
|Button.Square.chromium.png|0| Added|
|Button.Subtle Dark Mode.chromium.png|0| Added|
|Button.Subtle Disabled Dark Mode.chromium.png|0| Added|
|Button.Subtle Disabled Focusable Dark Mode.chromium.png|0| Added|
|Button.Subtle Disabled Focusable.chromium.png|0| Added|
|Button.Subtle Disabled.chromium.png|0| Added|
|Button.Subtle.chromium.png|0| Added|
|Button.Transparent Dark Mode.chromium.png|0| Added|
|Button.Transparent Disabled Dark Mode.chromium.png|0| Added|
|Button.Transparent Disabled Focusable Dark Mode.chromium.png|0| Added|
|Button.Transparent Disabled Focusable.chromium.png|0| Added|
|Button.Transparent Disabled.chromium.png|0| Added|
|Button.Transparent.chromium.png|0| Added|
|Button.With Long Text RTL.chromium.png|0| Added|
|Button.With Long Text.chromium.png|0| Added|
|Button.Circular.default.chromium.png|0| Removed|
|Button.Circular.hover.chromium.png|0| Removed|
|Button.Circular.pressed.chromium.png|0| Removed|
|Button.Default.default.chromium.png|0| Removed|
|Button.Default.hover.chromium.png|0| Removed|
|Button.Default.pressed.chromium.png|0| Removed|
|Button.Large Icon Only - Dark Mode.default.chromium.png|0| Removed|
|Button.Large Icon Only - Dark Mode.hover.chromium.png|0| Removed|
|Button.Large Icon Only - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Large Icon Only.default.chromium.png|0| Removed|
|Button.Large Icon Only.hover.chromium.png|0| Removed|
|Button.Large Icon Only.pressed.chromium.png|0| Removed|
|Button.Large With Icon - Dark Mode.default.chromium.png|0| Removed|
|Button.Large With Icon - Dark Mode.hover.chromium.png|0| Removed|
|Button.Large With Icon - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Large With Icon - RTL.default.chromium.png|0| Removed|
|Button.Large With Icon - RTL.hover.chromium.png|0| Removed|
|Button.Large With Icon - RTL.pressed.chromium.png|0| Removed|
|Button.Large With Icon.default.chromium.png|0| Removed|
|Button.Large With Icon.hover.chromium.png|0| Removed|
|Button.Large With Icon.pressed.chromium.png|0| Removed|
|Button.Large.default.chromium.png|0| Removed|
|Button.Large.hover.chromium.png|0| Removed|
|Button.Large.pressed.chromium.png|0| Removed|
|Button.Medium Icon Only - Dark Mode.default.chromium.png|0| Removed|
|Button.Medium Icon Only - Dark Mode.hover.chromium.png|0| Removed|
|Button.Medium Icon Only - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Medium Icon Only.default.chromium.png|0| Removed|
|Button.Medium Icon Only.hover.chromium.png|0| Removed|
|Button.Medium Icon Only.pressed.chromium.png|0| Removed|
|Button.Medium With Icon - Dark Mode.default.chromium.png|0| Removed|
|Button.Medium With Icon - Dark Mode.hover.chromium.png|0| Removed|
|Button.Medium With Icon - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Medium With Icon - RTL.default.chromium.png|0| Removed|
|Button.Medium With Icon - RTL.hover.chromium.png|0| Removed|
|Button.Medium With Icon - RTL.pressed.chromium.png|0| Removed|
|Button.Medium With Icon.default.chromium.png|0| Removed|
|Button.Medium With Icon.hover.chromium.png|0| Removed|
|Button.Medium With Icon.pressed.chromium.png|0| Removed|
|Button.Medium.default.chromium.png|0| Removed|
|Button.Medium.hover.chromium.png|0| Removed|
|Button.Medium.pressed.chromium.png|0| Removed|
|Button.Outline - Dark Mode.default.chromium.png|0| Removed|
|Button.Outline - Dark Mode.hover.chromium.png|0| Removed|
|Button.Outline - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Outline Disabled - Dark Mode.default.chromium.png|0| Removed|
|Button.Outline Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button.Outline Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Outline Disabled Focusable - Dark Mode.default.chromium.png|0| Removed|
|Button.Outline Disabled Focusable - Dark Mode.hover.chromium.png|0| Removed|
|Button.Outline Disabled Focusable - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Outline Disabled Focusable.default.chromium.png|0| Removed|
|Button.Outline Disabled Focusable.hover.chromium.png|0| Removed|
|Button.Outline Disabled Focusable.pressed.chromium.png|0| Removed|
|Button.Outline Disabled.default.chromium.png|0| Removed|
|Button.Outline Disabled.hover.chromium.png|0| Removed|
|Button.Outline Disabled.pressed.chromium.png|0| Removed|
|Button.Outline.default.chromium.png|0| Removed|
|Button.Outline.hover.chromium.png|0| Removed|
|Button.Outline.pressed.chromium.png|0| Removed|
|Button.Primary - Dark Mode.default.chromium.png|0| Removed|
|Button.Primary - Dark Mode.hover.chromium.png|0| Removed|
|Button.Primary - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Primary Disabled - Dark Mode.default.chromium.png|0| Removed|
|Button.Primary Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button.Primary Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Primary Disabled Focusable - Dark Mode.default.chromium.png|0| Removed|
|Button.Primary Disabled Focusable - Dark Mode.hover.chromium.png|0| Removed|
|Button.Primary Disabled Focusable - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Primary Disabled Focusable.default.chromium.png|0| Removed|
|Button.Primary Disabled Focusable.hover.chromium.png|0| Removed|
|Button.Primary Disabled Focusable.pressed.chromium.png|0| Removed|
|Button.Primary Disabled.default.chromium.png|0| Removed|
|Button.Primary Disabled.hover.chromium.png|0| Removed|
|Button.Primary Disabled.pressed.chromium.png|0| Removed|
|Button.Primary.default.chromium.png|0| Removed|
|Button.Primary.hover.chromium.png|0| Removed|
|Button.Primary.pressed.chromium.png|0| Removed|
|Button.Rounded.default.chromium.png|0| Removed|
|Button.Rounded.hover.chromium.png|0| Removed|
|Button.Rounded.pressed.chromium.png|0| Removed|
|Button.Small Icon Only - Dark Mode.default.chromium.png|0| Removed|
|Button.Small Icon Only - Dark Mode.hover.chromium.png|0| Removed|
|Button.Small Icon Only - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Small Icon Only.default.chromium.png|0| Removed|
|Button.Small Icon Only.hover.chromium.png|0| Removed|
|Button.Small Icon Only.pressed.chromium.png|0| Removed|
|Button.Small With Icon - Dark Mode.default.chromium.png|0| Removed|
|Button.Small With Icon - Dark Mode.hover.chromium.png|0| Removed|
|Button.Small With Icon - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Small With Icon - RTL.default.chromium.png|0| Removed|
|Button.Small With Icon - RTL.hover.chromium.png|0| Removed|
|Button.Small With Icon - RTL.pressed.chromium.png|0| Removed|
|Button.Small With Icon.default.chromium.png|0| Removed|
|Button.Small With Icon.hover.chromium.png|0| Removed|
|Button.Small With Icon.pressed.chromium.png|0| Removed|
|Button.Small.default.chromium.png|0| Removed|
|Button.Small.hover.chromium.png|0| Removed|
|Button.Small.pressed.chromium.png|0| Removed|
|Button.Square.default.chromium.png|0| Removed|
|Button.Square.hover.chromium.png|0| Removed|
|Button.Square.pressed.chromium.png|0| Removed|
|Button.Subtle - Dark Mode.default.chromium.png|0| Removed|
|Button.Subtle - Dark Mode.hover.chromium.png|0| Removed|
|Button.Subtle - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Subtle Disabled - Dark Mode.default.chromium.png|0| Removed|
|Button.Subtle Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button.Subtle Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Subtle Disabled Focusable - Dark Mode.default.chromium.png|0| Removed|
|Button.Subtle Disabled Focusable - Dark Mode.hover.chromium.png|0| Removed|
|Button.Subtle Disabled Focusable - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Subtle Disabled Focusable.default.chromium.png|0| Removed|
|Button.Subtle Disabled Focusable.hover.chromium.png|0| Removed|
|Button.Subtle Disabled Focusable.pressed.chromium.png|0| Removed|
|Button.Subtle Disabled.default.chromium.png|0| Removed|
|Button.Subtle Disabled.hover.chromium.png|0| Removed|
|Button.Subtle Disabled.pressed.chromium.png|0| Removed|
|Button.Subtle.default.chromium.png|0| Removed|
|Button.Subtle.hover.chromium.png|0| Removed|
|Button.Subtle.pressed.chromium.png|0| Removed|
|Button.Transparent - Dark Mode.default.chromium.png|0| Removed|
|Button.Transparent - Dark Mode.hover.chromium.png|0| Removed|
|Button.Transparent - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Transparent Disabled - Dark Mode.default.chromium.png|0| Removed|
|Button.Transparent Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Button.Transparent Disabled - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Transparent Disabled Focusable - Dark Mode.default.chromium.png|0| Removed|
|Button.Transparent Disabled Focusable - Dark Mode.hover.chromium.png|0| Removed|
|Button.Transparent Disabled Focusable - Dark Mode.pressed.chromium.png|0| Removed|
|Button.Transparent Disabled Focusable.default.chromium.png|0| Removed|
|Button.Transparent Disabled Focusable.hover.chromium.png|0| Removed|
|Button.Transparent Disabled Focusable.pressed.chromium.png|0| Removed|
|Button.Transparent Disabled.default.chromium.png|0| Removed|
|Button.Transparent Disabled.hover.chromium.png|0| Removed|
|Button.Transparent Disabled.pressed.chromium.png|0| Removed|
|Button.Transparent.default.chromium.png|0| Removed|
|Button.Transparent.hover.chromium.png|0| Removed|
|Button.Transparent.pressed.chromium.png|0| Removed|
|Button.With Long Text - RTL.default.chromium.png|0| Removed|
|Button.With Long Text - RTL.hover.chromium.png|0| Removed|
|Button.With Long Text - RTL.pressed.chromium.png|0| Removed|
|Button.With Long Text.default.chromium.png|0| Removed|
|Button.With Long Text.hover.chromium.png|0| Removed|
|Button.With Long Text.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>Checkbox</b> 48 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checked Dark Mode.chromium.png|0| Added|
|Checkbox.Checked.chromium.png|0| Added|
|Checkbox.Circular Checked Dark Mode.chromium.png|0| Added|
|Checkbox.Circular Checked.chromium.png|0| Added|
|Checkbox.Circular Dark Mode.chromium.png|0| Added|
|Checkbox.Circular Disabled Checked Dark Mode.chromium.png|0| Added|
|Checkbox.Circular Disabled Checked.chromium.png|0| Added|
|Checkbox.Circular Disabled Dark Mode.chromium.png|0| Added|
|Checkbox.Circular Disabled.chromium.png|0| Added|
|Checkbox.Circular Indeterminate Dark Mode.chromium.png|0| Added|
|Checkbox.Circular Indeterminate.chromium.png|0| Added|
|Checkbox.Circular RTL.chromium.png|0| Added|
|Checkbox.Circular.chromium.png|0| Added|
|Checkbox.Default Dark Mode.chromium.png|0| Added|
|Checkbox.Default RTL.chromium.png|0| Added|
|Checkbox.Default.chromium.png|0| Added|
|Checkbox.Disabled Checked Dark Mode.chromium.png|0| Added|
|Checkbox.Disabled Checked.chromium.png|0| Added|
|Checkbox.Disabled Dark Mode.chromium.png|0| Added|
|Checkbox.Disabled.chromium.png|0| Added|
|Checkbox.Indeterminate Dark Mode.chromium.png|0| Added|
|Checkbox.Indeterminate.chromium.png|0| Added|
|Checkbox.Checked - Dark Mode.normal.chromium.png|0| Removed|
|Checkbox.Checked.normal.chromium.png|0| Removed|
|Checkbox.Circular - Dark Mode.normal.chromium.png|0| Removed|
|Checkbox.Circular - RTL.normal.chromium.png|0| Removed|
|Checkbox.Circular Checked - Dark Mode.normal.chromium.png|0| Removed|
|Checkbox.Circular Checked.normal.chromium.png|0| Removed|
|Checkbox.Circular Disabled - Dark Mode.normal.chromium.png|0| Removed|
|Checkbox.Circular Disabled Checked - Dark Mode.normal.chromium.png|0| Removed|
|Checkbox.Circular Disabled Checked.normal.chromium.png|0| Removed|
|Checkbox.Circular Disabled.normal.chromium.png|0| Removed|
|Checkbox.Circular Indeterminate - Dark Mode.indeterminate.chromium.png|0| Removed|
|Checkbox.Circular Indeterminate - Dark Mode.normal.chromium.png|0| Removed|
|Checkbox.Circular Indeterminate.indeterminate.chromium.png|0| Removed|
|Checkbox.Circular Indeterminate.normal.chromium.png|0| Removed|
|Checkbox.Circular.normal.chromium.png|0| Removed|
|Checkbox.Default - Dark Mode.normal.chromium.png|0| Removed|
|Checkbox.Default - RTL.normal.chromium.png|0| Removed|
|Checkbox.Default.normal.chromium.png|0| Removed|
|Checkbox.Disabled - Dark Mode.normal.chromium.png|0| Removed|
|Checkbox.Disabled Checked - Dark Mode.normal.chromium.png|0| Removed|
|Checkbox.Disabled Checked.normal.chromium.png|0| Removed|
|Checkbox.Disabled.normal.chromium.png|0| Removed|
|Checkbox.Indeterminate - Dark Mode.indeterminate.chromium.png|0| Removed|
|Checkbox.Indeterminate - Dark Mode.normal.chromium.png|0| Removed|
|Checkbox.Indeterminate.indeterminate.chromium.png|0| Removed|
|Checkbox.Indeterminate.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Divider</b> 40 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider.Align Content Center.chromium.png|0| Added|
|Divider.Align Content End RTL.chromium.png|0| Added|
|Divider.Align Content End.chromium.png|0| Added|
|Divider.Align Content Start RTL.chromium.png|0| Added|
|Divider.Align Content Start.chromium.png|0| Added|
|Divider.Appearance Brand Dark Mode.chromium.png|0| Added|
|Divider.Appearance Brand.chromium.png|0| Added|
|Divider.Appearance Default Dark Mode.chromium.png|0| Added|
|Divider.Appearance Default.chromium.png|0| Added|
|Divider.Appearance Strong Dark Mode.chromium.png|0| Added|
|Divider.Appearance Strong.chromium.png|0| Added|
|Divider.Appearance Subtle Dark Mode.chromium.png|0| Added|
|Divider.Appearance Subtle.chromium.png|0| Added|
|Divider.Default Dark Mode.chromium.png|0| Added|
|Divider.Default.chromium.png|0| Added|
|Divider.Inset.chromium.png|0| Added|
|Divider.Vertical With Svg.chromium.png|0| Added|
|Divider.Vertical.chromium.png|0| Added|
|Divider.With Svg Rtl.chromium.png|0| Added|
|Divider.With Svg.chromium.png|0| Added|
|Divider.Align Content Center.normal.chromium.png|0| Removed|
|Divider.Align Content End - RTL.normal.chromium.png|0| Removed|
|Divider.Align Content End.normal.chromium.png|0| Removed|
|Divider.Align Content Start - RTL.normal.chromium.png|0| Removed|
|Divider.Align Content Start.normal.chromium.png|0| Removed|
|Divider.Appearance Brand - Dark Mode.normal.chromium.png|0| Removed|
|Divider.Appearance Brand.normal.chromium.png|0| Removed|
|Divider.Appearance Default - Dark Mode.normal.chromium.png|0| Removed|
|Divider.Appearance Default.normal.chromium.png|0| Removed|
|Divider.Appearance Strong - Dark Mode.normal.chromium.png|0| Removed|
|Divider.Appearance Strong.normal.chromium.png|0| Removed|
|Divider.Appearance Subtle - Dark Mode.normal.chromium.png|0| Removed|
|Divider.Appearance Subtle.normal.chromium.png|0| Removed|
|Divider.Default - Dark Mode.normal.chromium.png|0| Removed|
|Divider.Default.normal.chromium.png|0| Removed|
|Divider.Inset.normal.chromium.png|0| Removed|
|Divider.Vertical With Svg.normal.chromium.png|0| Removed|
|Divider.Vertical.normal.chromium.png|0| Removed|
|Divider.With Svg - RTL.normal.chromium.png|0| Removed|
|Divider.With Svg.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Label</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label.Default Dark Mode.chromium.png|0| Added|
|Label.Default.chromium.png|0| Added|
|Label.Disabled Dark Mode.chromium.png|0| Added|
|Label.Disabled.chromium.png|0| Added|
|Label.Required Dark Mode.chromium.png|0| Added|
|Label.Required RTL.chromium.png|0| Added|
|Label.Required.chromium.png|0| Added|
|Label.Size Large.chromium.png|0| Added|
|Label.Size Medium.chromium.png|0| Added|
|Label.Size Small.chromium.png|0| Added|
|Label.Weight Regular.chromium.png|0| Added|
|Label.Weight Semibold.chromium.png|0| Added|
|Label.Default - Dark Mode.normal.chromium.png|0| Removed|
|Label.Default.normal.chromium.png|0| Removed|
|Label.Disabled - Dark Mode.normal.chromium.png|0| Removed|
|Label.Disabled.normal.chromium.png|0| Removed|
|Label.Required - Dark Mode.normal.chromium.png|0| Removed|
|Label.Required - RTL.normal.chromium.png|0| Removed|
|Label.Required.normal.chromium.png|0| Removed|
|Label.Size Large.normal.chromium.png|0| Removed|
|Label.Size Medium.normal.chromium.png|0| Removed|
|Label.Size Small.normal.chromium.png|0| Removed|
|Label.Weight Regular.normal.chromium.png|0| Removed|
|Label.Weight Semibold.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Link</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link.Default.chromium.png|0| Added|
|Link.Inline.chromium.png|0| Added|
|Link.Subtle.chromium.png|0| Added|
|Link.With Long Text.chromium.png|0| Added|
|Link.Default.default.chromium.png|0| Removed|
|Link.Default.hover.chromium.png|0| Removed|
|Link.Default.pressed.chromium.png|0| Removed|
|Link.Inline.default.chromium.png|0| Removed|
|Link.Inline.hover.chromium.png|0| Removed|
|Link.Inline.pressed.chromium.png|0| Removed|
|Link.Subtle.default.chromium.png|0| Removed|
|Link.Subtle.hover.chromium.png|0| Removed|
|Link.Subtle.pressed.chromium.png|0| Removed|
|Link.With Long Text.default.chromium.png|0| Removed|

</details>


  <details><summary><b>MenuList</b> 58 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MenuList.Checkbox Dark Mode.chromium.png|0| Added|
|MenuList.Checkbox RTL.chromium.png|0| Added|
|MenuList.Checkbox With Icons RTL.chromium.png|0| Added|
|MenuList.Checkbox With Icons.chromium.png|0| Added|
|MenuList.Checkbox.chromium.png|0| Added|
|MenuList.Default Dark Mode.chromium.png|0| Added|
|MenuList.Default RTL.chromium.png|0| Added|
|MenuList.Default.chromium.png|0| Added|
|MenuList.Radio Dark Mode.chromium.png|0| Added|
|MenuList.Radio RTL.chromium.png|0| Added|
|MenuList.Radio With Icons RTL.chromium.png|0| Added|
|MenuList.Radio With Icons.chromium.png|0| Added|
|MenuList.Radio.chromium.png|0| Added|
|MenuList.With Icons.chromium.png|0| Added|
|MenuList.With Submenu Dark Mode.chromium.png|0| Added|
|MenuList.With Submenu RLT.chromium.png|0| Added|
|MenuList.With Submenu.chromium.png|0| Added|
|MenuList.Checkbox - Dark Mode.1st selected.chromium.png|0| Removed|
|MenuList.Checkbox - Dark Mode.2nd selected.chromium.png|0| Removed|
|MenuList.Checkbox - Dark Mode.normal.chromium.png|0| Removed|
|MenuList.Checkbox - RTL.1st selected.chromium.png|0| Removed|
|MenuList.Checkbox - RTL.2nd selected.chromium.png|0| Removed|
|MenuList.Checkbox - RTL.normal.chromium.png|0| Removed|
|MenuList.Checkbox With Icons - RTL.1st selected.chromium.png|0| Removed|
|MenuList.Checkbox With Icons - RTL.2nd selected.chromium.png|0| Removed|
|MenuList.Checkbox With Icons - RTL.normal.chromium.png|0| Removed|
|MenuList.Checkbox With Icons.1st selected.chromium.png|0| Removed|
|MenuList.Checkbox With Icons.2nd selected.chromium.png|0| Removed|
|MenuList.Checkbox With Icons.normal.chromium.png|0| Removed|
|MenuList.Checkbox.1st selected.chromium.png|0| Removed|
|MenuList.Checkbox.2nd selected.chromium.png|0| Removed|
|MenuList.Checkbox.normal.chromium.png|0| Removed|
|MenuList.Default - Dark Mode.hover menuitem.chromium.png|0| Removed|
|MenuList.Default - Dark Mode.normal.chromium.png|0| Removed|
|MenuList.Default - RTL.hover menuitem.chromium.png|0| Removed|
|MenuList.Default - RTL.normal.chromium.png|0| Removed|
|MenuList.Default.hover menuitem.chromium.png|0| Removed|
|MenuList.Default.normal.chromium.png|0| Removed|
|MenuList.Radio - Dark Mode.1st selected.chromium.png|0| Removed|
|MenuList.Radio - Dark Mode.2nd selected.chromium.png|0| Removed|
|MenuList.Radio - Dark Mode.normal.chromium.png|0| Removed|
|MenuList.Radio - RTL.1st selected.chromium.png|0| Removed|
|MenuList.Radio - RTL.2nd selected.chromium.png|0| Removed|
|MenuList.Radio - RTL.normal.chromium.png|0| Removed|
|MenuList.Radio With Icons - RTL.1st selected.chromium.png|0| Removed|
|MenuList.Radio With Icons - RTL.2nd selected.chromium.png|0| Removed|
|MenuList.Radio With Icons - RTL.normal.chromium.png|0| Removed|
|MenuList.Radio With Icons.1st selected.chromium.png|0| Removed|
|MenuList.Radio With Icons.2nd selected.chromium.png|0| Removed|
|MenuList.Radio With Icons.normal.chromium.png|0| Removed|
|MenuList.Radio.1st selected.chromium.png|0| Removed|
|MenuList.Radio.2nd selected.chromium.png|0| Removed|
|MenuList.Radio.normal.chromium.png|0| Removed|
|MenuList.With Icons.hover menuitem.chromium.png|0| Removed|
|MenuList.With Icons.normal.chromium.png|0| Removed|
|MenuList.With Submenu - Dark Mode.normal.chromium.png|0| Removed|
|MenuList.With Submenu - RTL.normal.chromium.png|0| Removed|
|MenuList.With Submenu.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>ProgressBar</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ProgressBar.Default Dark Mode.chromium.png|0| Added|
|ProgressBar.Default RTL.chromium.png|0| Added|
|ProgressBar.Default.chromium.png|0| Added|
|ProgressBar.Shape Rounded.chromium.png|0| Added|
|ProgressBar.Shape Square.chromium.png|0| Added|
|ProgressBar.Thickness Large.chromium.png|0| Added|
|ProgressBar.Thickness Medium.chromium.png|0| Added|
|ProgressBar.Validation State Error Dark Mode.chromium.png|0| Added|
|ProgressBar.Validation State Error.chromium.png|0| Added|
|ProgressBar.Validation State Success Dark Mode.chromium.png|0| Added|
|ProgressBar.Validation State Success.chromium.png|0| Added|
|ProgressBar.Validation State Warning Dark Mode.chromium.png|0| Added|
|ProgressBar.Validation State Warning.chromium.png|0| Added|
|ProgressBar.Value 0.chromium.png|0| Added|
|ProgressBar.Value 100.chromium.png|0| Added|
|ProgressBar.Value 25.chromium.png|0| Added|
|ProgressBar.Value 50.chromium.png|0| Added|
|ProgressBar.Value 75.chromium.png|0| Added|
|ProgressBar.Default - Dark Mode.normal.chromium.png|0| Removed|
|ProgressBar.Default - RTL.normal.chromium.png|0| Removed|
|ProgressBar.Default.normal.chromium.png|0| Removed|
|ProgressBar.Shape Rounded.normal.chromium.png|0| Removed|
|ProgressBar.Shape Square.normal.chromium.png|0| Removed|
|ProgressBar.Thickness Large.normal.chromium.png|0| Removed|
|ProgressBar.Thickness Medium.normal.chromium.png|0| Removed|
|ProgressBar.Validation State Error - Dark Mode.normal.chromium.png|0| Removed|
|ProgressBar.Validation State Error.normal.chromium.png|0| Removed|
|ProgressBar.Validation State Success - Dark Mode.normal.chromium.png|0| Removed|
|ProgressBar.Validation State Success.normal.chromium.png|0| Removed|
|ProgressBar.Validation State Warning - Dark Mode.normal.chromium.png|0| Removed|
|ProgressBar.Validation State Warning.normal.chromium.png|0| Removed|
|ProgressBar.Value 0.normal.chromium.png|0| Removed|
|ProgressBar.Value 100.normal.chromium.png|0| Removed|
|ProgressBar.Value 25.normal.chromium.png|0| Removed|
|ProgressBar.Value 50.normal.chromium.png|0| Removed|
|ProgressBar.Value 75.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Radio</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Radio.Checked Dark Mode.chromium.png|0| Added|
|Radio.Checked.chromium.png|0| Added|
|Radio.Default Dark Mode.chromium.png|0| Added|
|Radio.Default.chromium.png|0| Added|
|Radio.Disabled Checked Dark Mode.chromium.png|0| Added|
|Radio.Disabled Checked.chromium.png|0| Added|
|Radio.Disabled Dark Mode.chromium.png|0| Added|
|Radio.Disabled.chromium.png|0| Added|
|Radio.Checked - Dark Mode.normal.chromium.png|0| Removed|
|Radio.Checked.normal.chromium.png|0| Removed|
|Radio.Default - Dark Mode.normal.chromium.png|0| Removed|
|Radio.Default.normal.chromium.png|0| Removed|
|Radio.Disabled - Dark Mode.normal.chromium.png|0| Removed|
|Radio.Disabled Checked - Dark Mode.normal.chromium.png|0| Removed|
|Radio.Disabled Checked.normal.chromium.png|0| Removed|
|Radio.Disabled.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>RadioGroup</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RadioGroup.Default Checked.chromium.png|0| Added|
|RadioGroup.Default Dark Mode.chromium.png|0| Added|
|RadioGroup.Default RTL.chromium.png|0| Added|
|RadioGroup.Default.chromium.png|0| Added|
|RadioGroup.Disabled Dark Mode.chromium.png|0| Added|
|RadioGroup.Disabled Item Dark Mode.chromium.png|0| Added|
|RadioGroup.Disabled Item.chromium.png|0| Added|
|RadioGroup.Disabled.chromium.png|0| Added|
|RadioGroup.Horizontal Stacked RTL.chromium.png|0| Added|
|RadioGroup.Horizontal Stacked.chromium.png|0| Added|
|RadioGroup.Horizontal.chromium.png|0| Added|
|RadioGroup.Vertical RTL.chromium.png|0| Added|
|RadioGroup.Vertical.chromium.png|0| Added|
|RadioGroup.Default - Dark Mode.normal.chromium.png|0| Removed|
|RadioGroup.Default - RTL.normal.chromium.png|0| Removed|
|RadioGroup.Default Checked.normal.chromium.png|0| Removed|
|RadioGroup.Default.normal.chromium.png|0| Removed|
|RadioGroup.Disabled - Dark Mode.normal.chromium.png|0| Removed|
|RadioGroup.Disabled Item - Dark Mode.normal.chromium.png|0| Removed|
|RadioGroup.Disabled Item.normal.chromium.png|0| Removed|
|RadioGroup.Disabled.normal.chromium.png|0| Removed|
|RadioGroup.Horizontal Stacked - RTL.normal.chromium.png|0| Removed|
|RadioGroup.Horizontal Stacked.normal.chromium.png|0| Removed|
|RadioGroup.Horizontal.normal.chromium.png|0| Removed|
|RadioGroup.Vertical - RTL.normal.chromium.png|0| Removed|
|RadioGroup.Vertical.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Slider</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Default Dark Mode.chromium.png|0| Added|
|Slider.Default RTL.chromium.png|0| Added|
|Slider.Default.chromium.png|0| Added|
|Slider.Disabled Dark Mode.chromium.png|0| Added|
|Slider.Disabled.chromium.png|0| Added|
|Slider.Size Medium.chromium.png|0| Added|
|Slider.Size Small.chromium.png|0| Added|
|Slider.Slider Steps Vertical.chromium.png|0| Added|
|Slider.Slider Steps.chromium.png|0| Added|
|Slider.Vertical.chromium.png|0| Added|
|Slider.Default - Dark Mode.normal.chromium.png|0| Removed|
|Slider.Default - RTL.normal.chromium.png|0| Removed|
|Slider.Default.normal.chromium.png|0| Removed|
|Slider.Disabled - Dark Mode.normal.chromium.png|0| Removed|
|Slider.Disabled.normal.chromium.png|0| Removed|
|Slider.Size Medium.normal.chromium.png|0| Removed|
|Slider.Size Small.normal.chromium.png|0| Removed|
|Slider.Slider Steps Vertical.normal.chromium.png|0| Removed|
|Slider.Slider Steps.normal.chromium.png|0| Removed|
|Slider.Vertical.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Switch</b> 33 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch.Checked Dark.chromium.png|0| Added|
|Switch.Checked RTL.chromium.png|0| Added|
|Switch.Checked.chromium.png|0| Added|
|Switch.Default Dark.chromium.png|0| Added|
|Switch.Default RTL.chromium.png|0| Added|
|Switch.Default.chromium.png|0| Added|
|Switch.Disabled Checked Dark.chromium.png|0| Added|
|Switch.Disabled Checked.chromium.png|0| Added|
|Switch.Disabled Dark.chromium.png|0| Added|
|Switch.Disabled RTL.chromium.png|0| Added|
|Switch.Disabled.chromium.png|0| Added|
|Switch.Checked - Dark Mode.hover.chromium.png|0| Removed|
|Switch.Checked - Dark Mode.normal.chromium.png|0| Removed|
|Switch.Checked - RTL.hover.chromium.png|0| Removed|
|Switch.Checked - RTL.normal.chromium.png|0| Removed|
|Switch.Checked.hover.chromium.png|0| Removed|
|Switch.Checked.normal.chromium.png|0| Removed|
|Switch.Default - Dark Mode.hover.chromium.png|0| Removed|
|Switch.Default - Dark Mode.normal.chromium.png|0| Removed|
|Switch.Default - RTL.hover.chromium.png|0| Removed|
|Switch.Default - RTL.normal.chromium.png|0| Removed|
|Switch.Default.hover.chromium.png|0| Removed|
|Switch.Default.normal.chromium.png|0| Removed|
|Switch.Disabled - Dark Mode.hover.chromium.png|0| Removed|
|Switch.Disabled - Dark Mode.normal.chromium.png|0| Removed|
|Switch.Disabled - RTL.hover.chromium.png|0| Removed|
|Switch.Disabled - RTL.normal.chromium.png|0| Removed|
|Switch.Disabled Checked - Dark Mode.hover.chromium.png|0| Removed|
|Switch.Disabled Checked - Dark Mode.normal.chromium.png|0| Removed|
|Switch.Disabled Checked.hover.chromium.png|0| Removed|
|Switch.Disabled Checked.normal.chromium.png|0| Removed|
|Switch.Disabled.hover.chromium.png|0| Removed|
|Switch.Disabled.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Tabs</b> 59 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tabs.Appearance Subtle.chromium.png|0| Added|
|Tabs.Appearance Transparent.chromium.png|0| Added|
|Tabs.Default Dark Mode.chromium.png|0| Added|
|Tabs.Default RTL.chromium.png|0| Added|
|Tabs.Default.chromium.png|0| Added|
|Tabs.Disabled Dark Mode.chromium.png|0| Added|
|Tabs.Disabled.chromium.png|0| Added|
|Tabs.Horizontal.chromium.png|0| Added|
|Tabs.Navigation Over Disabled Item.chromium.png|0| Added|
|Tabs.Navigation RTL.chromium.png|0| Added|
|Tabs.Navigation.chromium.png|0| Added|
|Tabs.Size Large Dark Mode.chromium.png|0| Added|
|Tabs.Size Large Vertical Dark Mode.chromium.png|0| Added|
|Tabs.Size Large Vertical.chromium.png|0| Added|
|Tabs.Size Large.chromium.png|0| Added|
|Tabs.Size Medium Dark Mode.chromium.png|0| Added|
|Tabs.Size Medium Vertical Dark Mode.chromium.png|0| Added|
|Tabs.Size Medium Vertical.chromium.png|0| Added|
|Tabs.Size Medium.chromium.png|0| Added|
|Tabs.Size Small Dark Mode.chromium.png|0| Added|
|Tabs.Size Small Vertical Dark Mode.chromium.png|0| Added|
|Tabs.Size Small Vertical.chromium.png|0| Added|
|Tabs.Size Small.chromium.png|0| Added|
|Tabs.Vertical.chromium.png|0| Added|
|Tabs.Appearance Subtle.hover.chromium.png|0| Removed|
|Tabs.Appearance Subtle.normal.chromium.png|0| Removed|
|Tabs.Appearance Transparent.hover.chromium.png|0| Removed|
|Tabs.Appearance Transparent.normal.chromium.png|0| Removed|
|Tabs.Default - Dark Mode.normal.chromium.png|0| Removed|
|Tabs.Default - RTL.normal.chromium.png|0| Removed|
|Tabs.Default.normal.chromium.png|0| Removed|
|Tabs.Disabled - Dark Mode.normal.chromium.png|0| Removed|
|Tabs.Disabled.normal.chromium.png|0| Removed|
|Tabs.Horizontal.normal.chromium.png|0| Removed|
|Tabs.Navigation - RTL.2nd tab click.chromium.png|0| Removed|
|Tabs.Navigation - RTL.hover.chromium.png|0| Removed|
|Tabs.Navigation - RTL.normal.chromium.png|0| Removed|
|Tabs.Navigation - RTL.right arrow.chromium.png|0| Removed|
|Tabs.Navigation Over Disabled Item.2nd tab click.chromium.png|0| Removed|
|Tabs.Navigation Over Disabled Item.hover.chromium.png|0| Removed|
|Tabs.Navigation Over Disabled Item.normal.chromium.png|0| Removed|
|Tabs.Navigation Over Disabled Item.right arrow.chromium.png|0| Removed|
|Tabs.Navigation.2nd tab click.chromium.png|0| Removed|
|Tabs.Navigation.down arrow.chromium.png|0| Removed|
|Tabs.Navigation.hover.chromium.png|0| Removed|
|Tabs.Navigation.normal.chromium.png|0| Removed|
|Tabs.Size Large - Dark Mode.normal.chromium.png|0| Removed|
|Tabs.Size Large Vertical - Dark Mode.normal.chromium.png|0| Removed|
|Tabs.Size Large Vertical.normal.chromium.png|0| Removed|
|Tabs.Size Large.normal.chromium.png|0| Removed|
|Tabs.Size Medium - Dark Mode.normal.chromium.png|0| Removed|
|Tabs.Size Medium Vertical - Dark Mode.normal.chromium.png|0| Removed|
|Tabs.Size Medium Vertical.normal.chromium.png|0| Removed|
|Tabs.Size Medium.normal.chromium.png|0| Removed|
|Tabs.Size Small - Dark Mode.normal.chromium.png|0| Removed|
|Tabs.Size Small Vertical - Dark Mode.normal.chromium.png|0| Removed|
|Tabs.Size Small Vertical.normal.chromium.png|0| Removed|
|Tabs.Size Small.normal.chromium.png|0| Removed|
|Tabs.Vertical.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Text</b> 22 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Align.chromium.png|0| Added|
|Text.Block.chromium.png|0| Added|
|Text.Default.chromium.png|0| Added|
|Text.Font.chromium.png|0| Added|
|Text.Italic.chromium.png|0| Added|
|Text.Nowrap.chromium.png|0| Added|
|Text.Size.chromium.png|0| Added|
|Text.Strikethrough.chromium.png|0| Added|
|Text.Truncate.chromium.png|0| Added|
|Text.Underline.chromium.png|0| Added|
|Text.Weight.chromium.png|0| Added|
|Text.Align.normal.chromium.png|0| Removed|
|Text.Block.normal.chromium.png|0| Removed|
|Text.Default.normal.chromium.png|0| Removed|
|Text.Font.normal.chromium.png|0| Removed|
|Text.Italic.normal.chromium.png|0| Removed|
|Text.Nowrap.normal.chromium.png|0| Removed|
|Text.Size.normal.chromium.png|0| Removed|
|Text.Strikethrough.normal.chromium.png|0| Removed|
|Text.Truncate.normal.chromium.png|0| Removed|
|Text.Underline.normal.chromium.png|0| Removed|
|Text.Weight.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>TextInput</b> 56 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextInput.Appearance Filled Darker Dark Mode.chromium.png|0| Added|
|TextInput.Appearance Filled Darker.chromium.png|0| Added|
|TextInput.Appearance Filled Lighter Dark Mode.chromium.png|0| Added|
|TextInput.Appearance Filled Lighter.chromium.png|0| Added|
|TextInput.Appearance Outline Dark Mode.chromium.png|0| Added|
|TextInput.Appearance Outline.chromium.png|0| Added|
|TextInput.Appearance Underline Dark Mode.chromium.png|0| Added|
|TextInput.Appearance Underline.chromium.png|0| Added|
|TextInput.Content End Dark Mode.chromium.png|0| Added|
|TextInput.Content End RTL.chromium.png|0| Added|
|TextInput.Content End.chromium.png|0| Added|
|TextInput.Content Start Dark Mode.chromium.png|0| Added|
|TextInput.Content Start End Dark Mode.chromium.png|0| Added|
|TextInput.Content Start End RTL.chromium.png|0| Added|
|TextInput.Content Start End.chromium.png|0| Added|
|TextInput.Content Start RTL.chromium.png|0| Added|
|TextInput.Content Start.chromium.png|0| Added|
|TextInput.Default.chromium.png|0| Added|
|TextInput.Disabled Dark Mode.chromium.png|0| Added|
|TextInput.Disabled.chromium.png|0| Added|
|TextInput.Inline.chromium.png|0| Added|
|TextInput.Placeholder Dark Mode.chromium.png|0| Added|
|TextInput.Placeholder.chromium.png|0| Added|
|TextInput.Required Dark Mode.chromium.png|0| Added|
|TextInput.Required.chromium.png|0| Added|
|TextInput.Size Large.chromium.png|0| Added|
|TextInput.Size Medium.chromium.png|0| Added|
|TextInput.Size Small.chromium.png|0| Added|
|TextInput.Appearance Filled Darker - Dark Mode.normal.chromium.png|0| Removed|
|TextInput.Appearance Filled Darker.normal.chromium.png|0| Removed|
|TextInput.Appearance Filled Lighter - Dark Mode.normal.chromium.png|0| Removed|
|TextInput.Appearance Filled Lighter.normal.chromium.png|0| Removed|
|TextInput.Appearance Outline - Dark Mode.normal.chromium.png|0| Removed|
|TextInput.Appearance Outline.normal.chromium.png|0| Removed|
|TextInput.Appearance Underline - Dark Mode.normal.chromium.png|0| Removed|
|TextInput.Appearance Underline.normal.chromium.png|0| Removed|
|TextInput.Content End - Dark Mode.normal.chromium.png|0| Removed|
|TextInput.Content End - RTL.normal.chromium.png|0| Removed|
|TextInput.Content End.normal.chromium.png|0| Removed|
|TextInput.Content Start - Dark Mode.normal.chromium.png|0| Removed|
|TextInput.Content Start - RTL.normal.chromium.png|0| Removed|
|TextInput.Content Start End - Dark Mode.normal.chromium.png|0| Removed|
|TextInput.Content Start End - RTL.normal.chromium.png|0| Removed|
|TextInput.Content Start End.normal.chromium.png|0| Removed|
|TextInput.Content Start.normal.chromium.png|0| Removed|
|TextInput.Default.normal.chromium.png|0| Removed|
|TextInput.Disabled - Dark Mode.normal.chromium.png|0| Removed|
|TextInput.Disabled.normal.chromium.png|0| Removed|
|TextInput.Inline.normal.chromium.png|0| Removed|
|TextInput.Placeholder - Dark Mode.normal.chromium.png|0| Removed|
|TextInput.Placeholder.normal.chromium.png|0| Removed|
|TextInput.Required - Dark Mode.normal.chromium.png|0| Removed|
|TextInput.Required.normal.chromium.png|0| Removed|
|TextInput.Size Large.normal.chromium.png|0| Removed|
|TextInput.Size Medium.normal.chromium.png|0| Removed|
|TextInput.Size Small.normal.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>